contributor_docs/es/contributor_guidelines.md

```diff
@@ -0,0 +1,62 @@
+
+### Usando la interfaz de línea de comandos de `git`
+
+Cuando hayas realizado el <em>fork</em> (copia), navega a la página de tu <em>fork</em> y copia la URL de git haciendo clic en el botón verde que dice “Code”. Debería ser algo así como `https://github.com/limzykenneth/p5.js.git`
```

Comment:
para todas las palabras que queden en inglés por favor usar itálicas en lugar de em, como se mencionó en el grupo, aplica esto para todo el texto, omitiré los siguientes comentario pero se entiende que todos necesitan ser cambiados
CC: @Guirdo